certified-connectors/HrFlow/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1012 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "HrFlow.ai",
+    "description": "HrFlow.ai is the 1st multi-layer AI powered API that brings intelligence to your HR Data and bridges the gap between all your data Sources and Destinations.",
+    "contact": {
+        "name" : "HrFlow.ai",
+        "url" : "https://www.hrflow.ai",
+        "email" : "support@riminder.net"
+      }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.hrflow.ai"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://security.hrflow.ai/"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "AI;Website"
+    }
+  ],
+  "host": "api.hrflow.ai",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/v1/profile/parsing/file": {
+      "post": {
+        "summary": "This operation sends a document Profile for either sync or async parsing",
```

Comment:
@alaeddinechhima This summary value sounds like a description rather than a summary. Can you please improve it something like below:

` "summary": "Send a document Profile for either sync or async parsing"`